ibm/service/power/resource_ibm_pi_volume_clone.go

```diff
@@ -135,6 +143,10 @@ func resourceIBMPIVolumeCloneCreate(ctx context.Context, d *schema.ResourceData,
 		body.TargetReplicationEnabled = flex.PtrToBool(d.Get(Arg_ReplicationEnabled).(bool))
 	}
 
+	if v, ok := d.GetOk(Arg_UserTags); ok {
```

Comment:
No need of `flex.UpdateGlobalTagsUsingCRN` as well?

Reply:
The CRN of the cloned volume is not available in the struct that is used in the create function, so the user tags are not modified